charts/zookeeper/values.yaml

```diff
@@ -1,4 +1,10 @@
+replicas: 3
```

Comment:
Nice! I found myself making this same change here: https://github.com/derekm/zookeeper-operator/commit/99fa88513f3e995283453f8f994d333f3cb714d8